docs/changelog/2141.misc.rst

```diff
@@ -0,0 +1 @@
+add libffi-7.dll to the hard-coded list of dlls for PyPy
```

Comment:
Should start with capital and end with a period.

Reply:
fixed